internal/wrapnix/wrapper.go

```diff
@@ -72,12 +72,50 @@ func CreateWrappers(ctx context.Context, devbox devboxer) error {
 	return createSymlinksForSupportDirs(devbox.ProjectDir())
 }
 
+// CreateDevboxSymlink creates a symlink to the devbox binary
+//
+// Needed because:
+//
+//  1. The bin-wrappers cannot invoke the devbox via the Launcher. The Launcher script
+//     invokes some coreutils commands that may themselves be installed by devbox
+//     and so be bin-wrappers. This causes an infinite loop.
+//     So, the bin-wrappers need to directly invoke the devbox binary.
+//
+//  2. The devbox binary's path will change when devbox is updated. This means
+//     that bin-wrappers may invoke older devbox binaries, which have different
+//     functionality. Annoyingly, they will also print "New devbox available" messages,
+//     even though the user has just updated devbox.
+//     So, the bin-wrappers need to use a symlink to the latest devbox binary. This
+//     symlink is updated when devbox is updated.
+func CreateDevboxSymlink() (string, error) {
+	curDir := xdg.CacheSubpath(filepath.Join("devbox", "current"))
```

Comment:
thoughts on making this `devbox/bin/current` instead? that way we keep it with all the other bins.

Reply:
yeah, I like that. Will update.